### ISSUE TITLE
Segfault trying to start a gamma calibration from dev-version MonitorCenter

### ISSUE THREAD

**Issue by tomwallis** [psychopy/psychopy#1691]
I'm running the development version of PsychoPy (v1.90.0) in a conda environment. Visual display scripts run no problem. I just made [a small change](https://github.com/psychopy/psychopy/pull/1690) to get our LS100 communicating under Linux. I run MonitorCenter from the command line:

```sh
python psychopy/monitors/MonitorCenter.py
```

which opens fine. It can find the photometer no problem. When I click "gamma calibration" it opens the dialogue box with options as normal, but then when I select OK, a Window opens very briefly before dumping out with the following:

```sh
Fatal Python error: (pygame parachute) Segmentation Fault
Aborted (core dumped)
``` 

Suggestions?

**Comment by tomwallis** [comment]
This segfault is also happening after successfully detecting a PR670 (see [this thread](https://github.com/psychopy/psychopy/issues/1698)). The Standalone (v1.85.0 I think) runs, except that the gamma calibration window is not fullscreen (but in a window) and also the actual stimulus doesn't occupy all that window, despite the option "fraction of window" being set to 1.

Any ideas?

**Comment by peircej** [comment]
No, not a clue, off the top of my head. Can you try running the 'calibTools.getLumSeries()` script instead of running from Monitor Center, so you can debug better where the problem is?
